src/test/java/core/basesyntax/dao/FruitsDaoImplTest.java

```diff
@@ -0,0 +1,53 @@
+package core.basesyntax.dao;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exceptions.NoFruitsException;
+import java.util.Map;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+public class FruitsDaoImplTest {
+    private FruitsDao fruitsDao = new FruitsDaoImpl();

```

Comment:
you should initialize such fields inside of method with @BeforeClass, the same for other test classes